packages/amplify-provider-awscloudformation/src/override-manager/amplify-factory-transform.ts

```diff
@@ -0,0 +1,31 @@
+import { IAmplifyResource, AmplifyCategoryTransform, $TSContext, Template } from 'amplify-cli-core';
+
+const categoryCfnTransformMap = (resourceName: string) => ({
```

Comment:
can we not maintain another map of the plugins here - and instead make it dynamic. 
This would not work if a 3rd party plugins with override logic since there won't be mapping for it out here. This is a major blocker.

Reply:
Changed the logic to import a function dynamically from the plugin which initiates the transformation process